### PR TITLE
Release 18.1.0

### PR DESCRIPTION
**Pull request by wbclark**
I believe this should wait for #666 and #667 to be merged first. @evgeni could you please confirm?

### CODE REVIEW

**Comment by evgeni** [comment]
I don't think we'll get #666 green trivially. And I have expected tests on #667 to fail differently than what they did.

I can spin up a 20.04 box with #667 applied, and if it does what it should we could merge that without the metadata change?

**Comment by ekohl** [comment]
https://github.com/theforeman/puppet-foreman_proxy/pull/669 should be good and ready to merge.

**Review by ekohl — CHANGES_REQUESTED** [review]
Please update this with what's currently merged.

**Comment by wbclark** [comment]
Updated with latest changelogs and new date

**Review by ekohl — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+[Full Changelog](https://github.com/theforeman/puppet-foreman_proxy/compare/18.0.0...18.1.0)
+
+**Implemented enhancements:**
+
+- Support Ubuntu 20.04 [\#669](https://github.com/theforeman/puppet-foreman_proxy/pull/669) ([ekohl](https://github.com/ekohl))
+- don't manage runner repo on Debian [\#664](https://github.com/theforeman/puppet-foreman_proxy/pull/664) ([evgeni](https://github.com/evgeni))
+
+**Fixed bugs:**
+
+- Allow puppet-puppet \< 16.0.0 [\#665](https://github.com/theforeman/puppet-foreman_proxy/pull/665) ([wbclark](https://github.com/wbclark))
```
Comment (ekohl) [inline CHANGELOG.md:14]: Feels more like an enhancement to me. Doesn't matter that much since we're doing a minor version bump anyway.

**Review by ekohl — APPROVED** [review]

**Pull request merged**